Team_Roster/app/routes.py

```diff
@@ -19,4 +19,4 @@ def roster(teamId):
 @app.route('/<teamId>/depthChart', methods=['GET'])
 def depthChart(teamId):
     year = request.args.get("year")
-    return ShowDepthChart(teamId, year)
\ No newline at end of file
+    return ShowDepthChart(teamId)
```

Comment:
why not pass in the year anymore, don't you need it to calculate the depth chart???

Reply:
I was getting it from the URL, but I went ahead and changed it